src/locale/localed-util.c

```diff
@@ -86,6 +164,19 @@ void context_clear(Context *c) {
         c->polkit_registry = bus_verify_polkit_async_registry_free(c->polkit_registry);
 };
 
+static X11Context *context_get_x11_context(Context *c) {
+        assert(c);
+
+        if (!x11_context_isempty(&c->x11_from_xorg))
+                return &c->x11_from_xorg;
+
+        return NULL;
+}
+
+X11Context *context_get_x11_context_safe(Context *c) {
+        return &ASSERT_PTR(c)->x11_from_xorg;
+}
```

Comment:
Initially this is introduced as a mostly trivial function. But in a later commit, it will be extended.
Here, `_safe` does not mean the safety about the input, but it always return non-NULL value.

Reply:
Maybe call it `context_ensure_x11_context`?